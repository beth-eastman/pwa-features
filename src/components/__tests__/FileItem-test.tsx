/**
 * Testing the FileItem component
 */
import * as React from 'react';
import FileItem from '../PWAFeatures/Features/FileItem';
import * as renderer from 'react-test-renderer';


/*
  ShallowRender.dive() will return <FileItem /> component
 */
const wrapper = renderer.create(
    <FileItem
      name={'Some File'}
      type={'File Type'}
      size={1}
      lastModifiedDate={'1 billion years ago'}
    />
);

// tests
describe('<FileItem />', () => {

  it('should match snapshot tests', () => {
    let tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
