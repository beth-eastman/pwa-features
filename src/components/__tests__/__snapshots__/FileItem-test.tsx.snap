// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileItem /> should match snapshot tests 1`] = `
<li
  className="fileItem"
  style={
    Object {
      "textAlign": "left",
    }
  }
>
  File Name: 
  Some File
  <br />
  Type: 
  File Type
  <br />
  Size: 
  1
   bytes
  <br />
  Last Modified Date:  
  1 billion years ago
  <br />
</li>
`;
